tasks/sast-java-sec-check.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sast-java-sec-check
+spec:
+  workspaces:
+    - name: workspace
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
+  params:
+    - name: PATH_CONTEXT
+      description: The path to your source code
+      type: string
+      default: .
+    - name: OUTPUT_FORMAT
+      description: the format of findsecbugs output
+      type: string
+      default: "sarif"
+    - name: OUTPUT_ONLY_ANALYZE
+      description: |
+        only analyze given classes and packages;
+        end with .* to indicate classes in a package, .- to indicate a package prefix
+      type: string
+      default: ""
+    - name: OPTIONAL_ARGS
```

Comment:
Is it planned that this parameters together with OUTPUT_FORMAT and OUTPUT_ONLY_ANALYZE are going to be defined in pipeline in a future?

Reply:
I think we don't know if OUTPUT_ONLY_ANALYZE and  OPTIONAL_ARGS will be used in pipeline or not in the future, but we just provide an option. They are not mandatory parameters, the task still works if the two parameters are not defined.